sonar-project.properties

```diff
@@ -0,0 +1,8 @@
+sonar.python.version=3
+sonar.projectKey=charm-prometheus-juju-exporter
+sonar.organization=bootstack-devops
+sonar.python.coverage.reportPaths=coverage.xml
```

Comment:
This path should be changed to `tests/unit/report/`. It would be better to put both html and xml reports there